docs/LTS-strategy.md

```diff
@@ -0,0 +1,38 @@
+# Express LTS Strategy
+
+## Goals
+
+* Maintain strict Node.js version support within a single Express major release.
+* Provide good ergonomics and developer experience for Express maintainers
+* Provide good experience for users who want to keep their Express projects that up-to-date
+* Avoid stagnation and blocking innovation that is good for the end users
+
+## Non-Goals
+
+* Only support the active LTS releases of Node.js
+* Support the widest possible set of Node.js versions. We aim to maintain a healthy balance between backwards compatibility and toolchain modernization.
```

Comment:
Love this section, especially this line! 🎉

Reply:
I'm not actually sure this makes sense - a healthy balance imo is the same thing as "widest possible set". If a node version prevents using modern toolchains, for example, then that's not a version that'd be included in that set.

Reply:
@ljharb I'll think about possible wording improvements here, but the intent is to emphasize that we only want to support a set of versions that do not contradict the rest of the goals defined. This is very important to make explicit in the context of Express specifically, as it has a long-standing tradition of preserving the compatibility to the detriment of everything else.

Reply:
+1 this is a great way to explain it

Reply:
I don’t think any of the detriments you refer to have anything to do with refusing to drop compat for node, but with other things - but i look forward to an alternative phrasing.

Reply:
> a healthy balance imo is the same thing as "widest possible set"

As other said, I disagree with the wording but maybe your meaning hinges on the definition we use for "possible"?

Like if we said "we want to ship diagnostic channels so it is not possible to support versions before them" (I am trying to make a hypnotically based on a reality from the past https://github.com/pillarjs/router/pull/96) does that fit your definition? We would drop the old node versions which dont support it in the next major so we could ship support without a lot of compatibility code.

Reply:
Yes, of course it fits my definition. If we need a feature that requires dropping a version, then that version is no longer possible to support.

I also especially like the implied path of "ship support with compat code, and then drop the compat code and the node versions that need it in the next major" - that's a *very* user-friendly way to do a major bump.